src/lib/migrations/routines/03-add-tools-in-settings.ts

```diff
@@ -0,0 +1,24 @@
+import type { Migration } from ".";
+import { collections } from "$lib/server/database";
+import { ObjectId } from "mongodb";
+
+const addToolsToSettings: Migration = {
+	_id: new ObjectId(3),
```

Comment:
shouldnt the id be similar lenght to other mkigrations like this one ?
https://github.com/huggingface/chat-ui/blob/dbc0ca76f290ad222f93da5dc7beb3966c011cfc/src/lib/migrations/routines/04-update-message-updates.ts#L149